src/scancodes/set3/README.md

```
@@ -0,0 +1,427 @@
+```
+
+### Pause/Break (finally simple!)
+```
+Pause Press:     62
+Pause Release:   F0 62
+Break Press:     62    (Ctrl+Pause, same key)
+Break Release:   F0 62
+```
+
+### Windows Key (single byte)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)